staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -344,6 +347,12 @@ func (r *Reflector) watchHandler(w watch.Interface, resourceVersion *string, err
 	// Stopping the watcher should be idempotent and if we return from this function there's no way
 	// we're coming back in with the same watch interface.
 	defer w.Stop()
+	// update metrics
+	defer func() {
+		r.metrics.numberOfItemsInWatch.Observe(float64(eventCount))
```

Comment:
Why do you say it's the same metric twice? I don't see other reference to numberOfItemsInWatch.